sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/changefeed/pkversion/IncrementalChangeFeedProcessorTest.java

```diff
@@ -131,6 +131,7 @@ public void readFeedDocumentsStartFromBeginning() throws InterruptedException {
                     .setLeasePrefix("TEST")
                     .setMaxItemCount(10)
                     .setStartFromBeginning(true)
+                    .setStartTime()
                     .setMaxScaleCount(0) // unlimited
```

Comment:
Why is this needed?

Reply:
good catch - forgot to revert the change as was testing something else